engine/runtime/src/test/java/org/enso/interpreter/test/SignatureTest.java

```diff
@@ -621,10 +657,9 @@ public void andConversions() throws Exception {
           self.dict.mul self.value that.value
 
     compute (a : Plus & Mul) (b : Plus & Mul) =
-      add (x:Plus) (y:Plus) = x+y
-      p = add a b
+      p = a+b
       m = a*b
-      add p m
+      p:Plus + m:Plus
```

Comment:
Simplifies requesting conversions into desired types by using inline expressions `p:Plus` and `m:Plus`. Simpler than defining `add (x:Plus) (y:Plus)` method and requesting conversion of its arguments.